ufersa/plataforma-sabia
# [WEB] FAQ

## Summary


Partial #531 

## Relevant technical choices



This issue implements only the front-end side of things to the FAQ feature.
We'll need to come back later when the API is ready.

## QA Steps


1. Open a technology details page and navigate to the FAQ tab
![image](https://user-images.githubusercontent.com/10481028/99884499-6d9af180-2c0d-11eb-869d-9a8cab2132f0.png)
2. Login as a researcher and verify that you have the questions along with the notification counter showing at the dropdown menu
![image](https://user-images.githubusercontent.com/10481028/99884535-bb175e80-2c0d-11eb-9c08-af91213e9a57.png)
![image](https://user-images.githubusercontent.com/10481028/99884569-03cf1780-2c0e-11eb-83fb-291fbede3c3d.png)
3. Navigate to the questions page and verify that all of them are being listed and can be viewed
![image](https://user-images.githubusercontent.com/10481028/99884576-10ec0680-2c0e-11eb-90ee-162484d715d9.png)
![image](https://user-images.githubusercontent.com/10481028/99884578-19dcd800-2c0e-11eb-9928-01707543e9b9.png)

## Checklist
